client/me/concierge/shared/available-time-card.js

```diff
@@ -161,22 +283,57 @@ class CalendarCard extends Component {
 				header={ this.renderHeader() }
 			>
 				<FormFieldset>
+					<FormLabel htmlFor="concierge-start-time-group">
+						{ translate( 'Choose time of day' ) }
+					</FormLabel>
+					<Button
+						className={ btnMorningTimeGroup }
+						onClick={ () => this.handleFilterClick( 'morning' ) }
+						disabled={ morningTimes.length === 0 }
+						title={ btnMorningTitle }
```

Comment:
I am not sure if I have seen the title attribute being used in a button tag. Could you help me understand the usage?

Reply:
Title attribute is allowed for <button /> tag. The reason it's used here is to provide more context when the "Morning" or "Afternoon" filter is disabled due to non-availability of slots.